tests/infra_test.go

```diff
@@ -1690,6 +1690,112 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", Serial, decorators.SigCo
 		})
 	})
 
+	Describe("virt-handler", func() {
+		var (
+			originalKubeVirt *v1.KubeVirt
+			nodeToEnableKSM  string
+		)
+
+		getNodesWithKSMAvailable := func(virtCli kubecli.KubevirtClient) []*k8sv1.Node {
+			nodes := libnode.GetAllSchedulableNodes(virtCli)
+
+			nodesWithKSM := make([]*k8sv1.Node, 0)
+			for _, node := range nodes.Items {
+				command := []string{"cat", "/sys/kernel/mm/ksm/run"}
+				_, err := tests.ExecuteCommandInVirtHandlerPod(node.Name, command)
+				if err == nil {
+					nodesWithKSM = append(nodesWithKSM, &node)
+				}
+			}
+			return nodesWithKSM
+		}
+
+		BeforeEach(func() {
+			nodesWithKSM := getNodesWithKSMAvailable(virtClient)
+			if len(nodesWithKSM) == 0 {
+				Skip("Skip testing with ksm, because there are no nodes with ksm available")
```

Comment:
Shouldn't we use Ginkgo labels/decorators instead of blindly skipping?

Reply:
I had thought about it but couldn't think of a way to activate that label. Now that you've pointed it out to me again, it's quite simple actually. Thanks

Reply:
Done